lib/s3.persist.stream.js

```diff
@@ -0,0 +1,127 @@
+var AWS = require('aws-sdk');
+var _ = require('underscore');
+
+var stream = require('stream');
+var util = require('util');
+
+var rotateHistoryCount = 3;
```

Comment:
i think ill leave this comment unresolved untill we talk with oschribe  about how to rotate